internal/e2e/e2e_test.go

```diff
@@ -202,20 +206,83 @@ func TestPublishEndpoint(t *testing.T) {
 		}
 
 		for _, k := range key.Keys {
-			gotExported[base64.StdEncoding.EncodeToString(k.KeyData)] = true
-		}
-		allExported := true
-		for _, want := range wantExport {
-			if _, ok := gotExported[want]; !ok {
-				allExported = false
+			sk := base64.StdEncoding.EncodeToString(k.KeyData)
+			if _, ok := wantExport[sk]; ok {
+				expectedFile = latest
+				gotExported[sk] = true
 			}
 		}
-		if !allExported {
+		if diff := cmp.Diff(wantExport, gotExported); diff != "" {
 			return retry.RetryableError(errors.New("Not all keys are exported yet, keep waiting"))
 		}
 
 		return nil
 	})
+
+	// Mark the export in the past to force a cleanup
+	t.Logf("Exposure was exported: %q. Modify the date to 14 days earlier", expectedFile)
+	exportFile, err := exportdb.New(db).LookupExportFile(ctx, expectedFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := db.InTx(ctx, pgx.Serializable, func(tx pgx.Tx) error {
+		result, err := tx.Exec(ctx, `
+		UPDATE
+			ExportBatch
+		SET
+			start_timestamp = $1,
+			end_timestamp = $2
+		WHERE
+			batch_id = $3
+	`,
+			time.Now().Add(-30*24*time.Hour),
+			time.Now().Add(-29*24*time.Hour),
```

Comment:
If you wanted, you could also use https://godoc.org/time#Time.AddDate, but this is fine.